appcenter-pre-build.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
```

Comment:
Is moving the file necessary for RN builds? 
If yes we should consider a way of flagging platform types to not do the extra work if not needed.

Reply:
Moving the files, yes. Flags for Platform types, AppCenter does not provide.